.github/workflows/docker-release.yml

```diff
@@ -1,10 +1,8 @@
-name: Build Docker Image
+name: Build Stable Docker Image
 
-# Docker Images are built for every push to master or when a new tag is created
+# Stable Docker images are built for every new tag
```

Comment:
Your comment update makes me think: maybe we ought to truly build for every new tag? Instead of just `v*.*.*`. After all, why would anyone with permission push a tag we didn't want someone to be able to try out? 🤔

Reply:
IMO we should keep the `:latest` channel to official releases (e.g. `vX.Y.Z`) so people don’t unintentionally get bumped to an unstable version. It might make sense to set that up in the future but we can always figure that out when we need it :)